### PR TITLE
Remove wrap on plugin logs table

### PR DESCRIPTION
**Pull request by mariusandra**
## Changes

- Removes `text-overflow:ellipsis` from the plugin logs table, as you just can't read all the information in a log with it.
- The resizing part seems to be slightly broken though, but I'd not put that in the scope of this PR.
- On a related note, [this](https://github.com/PostHog/posthog/blob/f75d25ad64377e4362222a9303ed609e23c627ae/frontend/src/lib/components/ResizableTable/index.scss#L20) seems pretty dangerous. It basically adds `text-overflow: ellipsis` to every table cell everywhere in posthog. It feels like a lot for a `lib/` util to do :). What should I do to opt out? cc @samwinslow 

- Before:

- After:

- At least you can read the logs now. Thus I'd really love to get this in to the release, unless a real fix is imminent :)

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious

### CODE REVIEW

**Review by Twixes — APPROVED** [review]
I made a real fix. :)
Overall agreed that such broad `text-overflow: ellipsis` may be… too broad.
fix

File: frontend/src/lib/components/ResizableTable/VirtualTableHeader.tsx [hunk 1]
```diff
@@ -66,13 +67,14 @@ function VirtualTableHeader<RecordType>({
...
     minColumnWidth,
+    expandable,
 }: VirtualTableHeaderProps<RecordType>): JSX.Element {
     const maxColumnWidth = minColumnWidth * 12
     const height = 60
     useLayoutEffect(() => (typeof layoutEffect === 'function' ? layoutEffect() : undefined))
     return (
         <div className="resizable-virtual-table-header">
-            <div className="left-spacer" style={{ width: ANTD_EXPAND_BUTTON_WIDTH }} />
+            {expandable && <div className="left-spacer" style={{ width: ANTD_EXPAND_BUTTON_WIDTH }} />}
```
Comment (samwinslow) [inline frontend/src/lib/components/ResizableTable/VirtualTableHeader.tsx:77]: This is good! It makes ResizableTable much more portable.

**Review by samwinslow — APPROVED** [review]
LGTM!

**Comment by mariusandra** [comment]
Much better, thanks @Twixes

However there's still something with max with:



Also, @samwinslow I'm really really not sure that it's a great idea to have a global "make every table cell everywhere have `white-space nowrap` and `text-overflow: ellipsis`. Is there any way to either only enable it for resizable tables by default... or provide some really easy way to opt-out, without having to add [some hacks](https://github.com/PostHog/posthog/pull/4246/files#diff-933fd189df1e21fbaccc269ce4bc078643a3d18fdda25ed159739e0db19539d6R67-R73) to a ".scss" file? This global rule is going to cause you trouble going forward, guaranteed :).

**Comment by samwinslow** [comment]
These defaults are definitely too opinionated! I'll get out a fix that removes `white-space` and `text-overflow` styles and sets them where necessary (Sessions, Persons, and Events, where the table was first implemented). Thanks for the feedback.

**Comment by mariusandra** [comment]
Awesome! Thanks for this :)

**Pull request merged**